src/dcp/dcp.c

```diff
@@ -231,6 +234,23 @@ int main(int argc, char** argv)
                     }
                 }
                 break;
+            case 'e':
+                copy_xattrs_used = 1;
+                if (!strcmp(optarg,"none")) {
```

Comment:
Not sure what the coding guidelines are for MPIFileutils, but (IMHO) the usage of "!strcmp()" can be misread, as it looks like it is checking "if not a good string compare".  For Lustre we prefer to use "strcmp() == 0", but by no means am I trying to push my code style upon you, just a thought.